README.md

```diff
@@ -1,5 +1,9 @@
 # DevOps-Apprenticeship-Notes
+k-6-youtube-workshops
+* Here is the docker post-install instructions that reduce the number of sudo commands: https://docs.docker.com/engine/install/linux-postinstall/
 
+*  adding some tfswitch vedio. Here is the tfswitch video link: https://drive.google.com/file/d/14xNJzpCajLEVzcGL5h3twfIyD-hRhkGc/view?usp=sharing
+=======
```

Comment:
Im guessing you had a merge conflict. Please remove the `===` as they are only useful in the process of resolving the merge conflict.

Reply:
Thank you Micheal. I fixed everything that you asked. let me know if anything needs to change.